src/ray/protobuf/experimental/autoscaler.proto

```diff
@@ -271,6 +275,33 @@ message GetClusterStatusReply {
   ClusterResourceState cluster_resource_state = 2;
 }
 
+enum DrainNodeReason {
+  // The node has been idle for long enough time.
+  // Drain request with this reason is rejectable.
+  IDLE_TERMINATION = 0;
```

Comment:
will we add more reason like 'compaction...'?

Reply:
Maybe, if needed.